stack-easybib/spec/role-nginxapp_spec.rb

```diff
@@ -88,21 +88,6 @@
             include("access_log #{access_log};")
           )
       end
-
-      describe 'pools' do
-        before do
-          node.override['php-fpm']['pools'] = %w(www1 www2 www3)
-        end
-
-        it 'creates three upstreams' do
```

Comment:
That is how I understood you:
```
@bwiborg·11:57 AM
@till [...] at least the tests all pass except 'creates three upstreams'
@till·12:31 PM
@bwiborg Doubt it we need that. Let's solve the problem at hand.
```

Thus, I removed that test and continued solving the problem at hand.